classes/form/group_render_context.php

```diff
@@ -130,7 +130,9 @@ public function set_default(string $name, $default): void {
      */
     public function add_rule(string  $name, ?string $message, string $type, ?string $format = null,
                              ?string $validation = "server", bool $reset = false, bool $force = false): void {
-        utils::ensure_exists($this->rules, $name)[] = [$message, $type, $format, $validation, $reset];
+        utils::ensure_exists($this->rules, form_name_mangler::mangle($name))[] = [
+            $message, $type, $format, $validation, $reset
```

Comment:
Müsste hier nicht noch `$force` mitgegeben werden?
```suggestion
            $message, $type, $format, $validation, $reset, $force
```

Reply:
Stimmt :+1: 
Also tatsächlich unterstützt `addGroupRule` anscheinend kein `force`, aber diese Klasse wird auch für die Repitition verwendet, die das sehr wohl tut.